TRANSLATING.md

```diff
@@ -0,0 +1,97 @@
+## Introduction
+
+It is possible to translate the website to other languages, and anyone is very welcome to do so.
+
+Note that we're taking translations very seriously.
+They should receive as much love as the English version.
+
+For example, Google Translate is already built into Google Chrome.
+Don't bother using software to create translations of the docs.
+If we create a translation of the docs for a new language, then because we want high quality translations written by native speakers or near-native speakers.
+Examples and real-world comparisons must be replaced by equivalents in the culture of the respective country.
+A good translation is hard and it takes time.
+We don't want partial or outdated translations.
+
+Anyone who wants to start a new language should ask themselves the following questions before putting any work into docs.
+
+1. How much of the documentation do you want to translate?
+   If the answer isn't to get to 100 % eventually, don't start.
+2. How long are you going to stick around in bot development?
+   grammY is young.
+   It evolves quickly and there are many articles that get changed often.
+   The translations must be kept in sync with the English original.
+   Outdated and wrong docs are much worse than no docs. Many people only come here for a few months (and that's fine!), but if you translate everything and then stop developing bots and stop updating the articles, we'll have to remove that language again.
+   Then it was a waste of time for you.
+   So if you don't want to commit long-term, don't start.
+3. How many hours per week can you spend in the beginning?
+   It's understandable that a first PR doesn't contain translations for all pages immediately.
+   However, if you can only work on it rarely, it's going to take a very long time until you're done.
+   We don't want to end up with 15 languages and none of them is done for months.
+   If you can't set aside enough free time to get this done, don't start.
+
+## The Translation Workflow
+
+Translating the grammY docs into a language happens in two stages: the Active Stage and the Maintenance Stage.
+
+When a language is first started, no translations are available.
+Hence, the complete website must be translated actively (Active Stage).
+This requires an insane amount of work and a fair bit of dedication.
+Do not underestimate the time you will need for this.
+Respect to everyone who manages to complete the work!
+
+The language will then enter Maintenance Stage.
+From here on, all that needs to be done is to keep notifications on for new pull requests, and sync the translations whenever an article gets updated.
+
+Here is some more information about how each stage works.
+
+### Active Stage
+
+A translation to another language must be 100% complete before it gets published.
+We do not accept partial translations.
+All menus, links, articles, comments, and everything translatable must be localised before the changes can finally go live on the website.
```

Comment:
We should also mention that people need to open a tracking issue. We assume this later. This was one of the recommendations in the old file.
```suggestion
All menus, links, articles, comments, and everything translatable must be localised before the changes can finally go live on the website.
Please open a translation tracking issue so everyone can follow the progress.
It can look like https://github.com/grammyjs/website/issues/55.
```